functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,128 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the lenght of a given argument in bits. It calculates the length of the argument and then returns the result in bits.
+
+Examples:
+
+```sql
+SELECT BIT_LENGTH("TiDB");
++----------+
+| BIT_LENGTH("TiDB") |
++----------+
+| 8 (T) + 8 (i) + 8 (D) + 8 (B) = 40 bits |
```

Comment:
This is mixing actual output with the description.

```
mysql> SELECT BIT_LENGTH("TiDB");
+--------------------+
| BIT_LENGTH("TiDB") |
+--------------------+
|                 32 |
+--------------------+
1 row in set (0.01 sec)
```

Also maybe it would be better to say: "8 bits per character × 4 characters = 40 bits"